service/controller/appcatalog/v1/resource/appcatalog/resource.go

```diff
@@ -0,0 +1,80 @@
+package appcatalog
```

Comment:
Lets rename this resource to `index`. We want to fetch the index.yaml for the catalog and save it to a configmap. This is so we can use it in our APIs and display in Happa. 
https://github.com/giantswarm/sample-catalog/blob/master/index.yaml 

For now we can just add the empty resource.

cc @puja108 @oponder

Reply:
I don't get that, in diagram that I saw it before, it doesn't have any CR named `index`.

Reply:
The CRD is appCatalog the resources are named after what the operator needs to do. In this case we need to save the index.yaml so calling it index makes sense.

@giantswarm/team-batman Chime in if you disagree.